Makefile

```diff
@@ -614,7 +576,7 @@ else
 		cp wapm-cli/$(TARGET_DIR)/wapm package/bin/ ;\
 	fi
 ifeq ($(IS_DARWIN), 1)
-	codesign -s - package/bin/wapm
+	# codesign -s - package/bin/wapm || true
```

Comment:
To be removed from this diff, I suppose?

Reply:
Yup, published by mistake. Good call!